src/lamplib/tasks/run_tests.py

```diff
@@ -110,7 +110,11 @@ def resmoke_test(env, suites, mongo_dir, is_cnats):
         # Default download location for MongoDB binaries.
         env["PATH"] += ":" + os.path.join(mongo_dir, "bin") + ":" + mongo_dir
 
-    evg_venv_dir = os.path.join(workdir, "build", "venv")
+    if 'LAMP_VENV_DIR' not in os.environ:
+        raise ValueError("The venv directory is required for resmoke and does not exist, "
```

Comment:
This error message is wrong and should say "The LAMP_VENV_DIR environment variable must be set to the parent directory of the path which contains the python virtual environment". Reuse this error message in the os.path.exists call mentioned elsewhere.

Reply:
LAMP_VENV_DIR is an internal concept hidden to genny users. Are you sure we want to mention it in the error message? :)